app/web_ui/src/routes/(app)/run/run_input_form.svelte

```diff
@@ -74,7 +94,8 @@
     <FormElement
       id={id + "_" + property.id}
       label={property.title}
-      inputType={property.type === "string" ? "textarea" : "input"}
+      inputType={get_input_type(property)}
+      info_description={get_info_description(property)}
```

Comment:
JSON scheme isn't very readable. People often confuse it for a template.

I'd keep it simple, something like "A list of items in JSON format. For example `[ITEM_1, ITEM_2]`" (literally just "ITEM_1", not something dynamic).